Sources/HipMobileUI/Resources/Strings.resx

```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
   </data>
 	<data name="LicenseScreenView_XSL_Title" xml:space="preserve">
     <value>Xamarin Support Libraries</value>
   </data>
 	<data name="MainPageViewModel_Feedback" xml:space="preserve">
     <value>Feedback</value>
   </data>
-	<data name="MainPageViewModel_LegalNotices" xml:space="preserve">
-    <value>Rechtliche Hinweise</value>
+  <data name="MainPageViewModel_LegalNotices" xml:space="preserve">
+    <value>Legal Notice</value>
   </data>
-	<data name="MainPageViewModel_OverviewPage" xml:space="preserve">
-    <value>Übersicht</value>
+  <data name="MainPageViewModel_OverviewPage" xml:space="preserve">
+    <value>Overview</value>
   </data>
-	<data name="MainPageViewModel_Routes" xml:space="preserve">
-    <value>Routen</value>
+  <data name="MainPageViewModel_Routes" xml:space="preserve">
+    <value>Routes</value>
   </data>
-	<data name="MainPageViewModel_Settings" xml:space="preserve">
-    <value>Einstellungen</value>
+  <data name="MainPageViewModel_Settings" xml:space="preserve">
+    <value>Settings</value>
   </data>
-	<data name="MainPage_MenuTitle" xml:space="preserve">
-    <value>Historisches Paderborn</value>
+  <data name="MainPage_MenuTitle" xml:space="preserve">
+    <value>Historical Paderborn</value>
   </data>
-	<data name="RouteDetailsPageViewModel_Description" xml:space="preserve">
-    <value>Beschreibung</value>
+  <data name="RouteDetailsPageViewModel_Description" xml:space="preserve">
+    <value>Description</value>
   </data>
 	<data name="RouteDetailsPageViewModel_Distance" xml:space="preserve">
     <value>{0} km</value>
   </data>
 	<data name="RouteDetailsPageViewModel_Duration" xml:space="preserve">
     <value>{0} min</value>
   </data>
-	<data name="RouteDetailsPageViewModel_Map" xml:space="preserve">
-    <value>Karte</value>
+  <data name="RouteDetailsPageViewModel_Map" xml:space="preserve">
+    <value>Map</value>
   </data>
-	<data name="RouteDetailsPage_PlayAudio" xml:space="preserve">
-    <value>Vorlesen</value>
+  <data name="RouteDetailsPage_PlayAudio" xml:space="preserve">
+    <value>Read aloud</value>
   </data>
-	<data name="RouteDetailsPage_PauseAudio" xml:space="preserve">
-    <value>Vorlesen pausieren</value>
+  <data name="RouteDetailsPage_PauseAudio" xml:space="preserve">
+    <value>Pause reading aloud</value>
   </data>
-	<data name="RouteDetailsPage_StartRoute" xml:space="preserve">
-    <value>Route starten</value>
+  <data name="RouteDetailsPage_StartRoute" xml:space="preserve">
+    <value>Start route</value>
   </data>
-	<data name="RouteDetailsPage_Tags" xml:space="preserve">
-    <value>Auf der Route</value>
+  <data name="RouteDetailsPage_Tags" xml:space="preserve">
+    <value>On the route</value>
   </data>
-	<data name="RoutesOverviewListItemViewModel_Distance" xml:space="preserve">
-    <value>{0} Kilometer</value>
+  <data name="RoutesOverviewListItemViewModel_Distance" xml:space="preserve">
+    <value>{0} kilometre</value>
```

Comment:
We are talking about the plural of _kilometre_ which imo should be _kilometres_.